external-crates/move/crates/move-compiler/src/shared/ide.rs

```diff
@@ -82,11 +82,11 @@ pub struct DotAutocompleteInfo {
 #[derive(Default, Debug, Clone)]
 pub struct AliasAutocompleteInfo {
     /// Numerical addresses that are valid autocompletes
-    pub addresses: BTreeSet<(Symbol, NumericalAddress)>,
+    pub addresses: BTreeMap<Symbol, NumericalAddress>,
     /// Modules that are valid autocompletes
-    pub modules: BTreeSet<(Symbol, E::ModuleIdent)>,
+    pub modules: BTreeMap<Symbol, E::ModuleIdent>,
     /// Members that are valid autocompletes
-    pub members: BTreeSet<(Symbol, E::ModuleIdent, Name)>,
+    pub members: BTreeMap<E::ModuleIdent, BTreeMap<Symbol, Name>>,
```

Comment:
Is there a good reason to change these over? It doesn't matter too much to me, I'm just curious.

Reply:
At some point, I thought I'd need to search this by-module and figured it would be a better structure for this. I ended up not needing this at all and reverted this change